src/views/SignBtcTransactionLedger.vue

```diff
@@ -122,51 +120,19 @@ export default class SignBtcTransactionLedger extends SignBtcTransaction {
     }
 
     protected async _signBtcTransaction(transactionInfo: BitcoinTransactionInfo, walletInfo: WalletInfo) {
```

Comment:
`walletInfo` unused?

Reply:
It's also to match the signature in the parent class.
Actually in this case I could omit the parameter, but I'll use it in the future anyways.